File: bench/bench.js
function fib_iter(n) {
	let current = 0;
	let next = 1;

	let i = 0;
	while (true) {
		if (i >= n) {
			break;
		}
		const new_ = next + current;
		current = next;
		next = new_;
		i = i + 1;
	}

	current;
}

const N = 99999; // 78 is max for result without deviation
console.log("fib_iter(", N, ") => ");
console.log(fib_iter(N));

File: bench/bench.py
import sys
sys.set_int_max_str_digits(9999999)


def fib_iter(n):
    current = 0
    next = 1

    i = 0
    while True:
        if i >= n:
            break

        new = next + current
        current = next
        next = new
        i = i + 1

    return current


N = 99999
print("fib_iter(", N, ") => ")
print(fib_iter(N))
